packages/malloy/src/model/malloy_query.ts

```diff
@@ -1929,7 +1931,18 @@ class QueryQuery extends QueryField {
 
   expandFields(resultStruct: FieldInstanceResult) {
     let resultIndex = 1;
-    for (const f of this.expandWildCards(resultStruct.firstSegment.fields)) {
+    let fields: QueryFieldDef[] = resultStruct.firstSegment.fields;
```

Comment:
Core premise of the PR is that we should be processing order by fields explicitly into the resultStruct, otherwise we only get them by chance if they are also included into the query via some other parameter